### PR TITLE
Use environment specific terraform roles

### PR DESCRIPTION
**Pull request by MancunianSam**
The terraform role was hard coded for integration.

### CODE REVIEW

File: common.tf [hunk 1]
```diff
@@ -94,7 +94,7 @@ module "dr2_kms_key" {
       module.ingest_folder_opex_creator_lambda.lambda_role_arn,
       module.ingest_upsert_archives_folder_lambda.lambda_role_arn
     ], local.additional_user_roles)
-    ci_roles      = ["arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/IntgTerraformRole"]
+    ci_roles      = ["arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/${title(local.environment)}TerraformRole"]
```
Comment (techncl) [inline common.tf:97]: Couldn't you use `local.environment_title` here instead?

File: common.tf [hunk 2]
```diff
@@ -107,7 +107,7 @@ module "dr2_developer_key" {
       data.aws_ssm_parameter.dev_admin_role.value,
       module.preservica_config_lambda.lambda_role_arn
     ]
-    ci_roles      = ["arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/IntgTerraformRole"]
+    ci_roles      = ["arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/${title(local.environment)}TerraformRole"]
```
Comment (techncl) [inline common.tf:110]: Couldn't you use `local.environment_title` here instead?

**Review by techncl — COMMENTED** [review]
Looks good! Just a minor comment

File: common.tf (hunk 1 above)
Comment (MancunianSam) [reply common.tf]: Good spot, I forgot about that. I've changed it. Same for the one below.

**Review by techncl — APPROVED** [review]

**Pull request merged**